src/main/java/org/alliancegenome/curation_api/base/entity/AuditedObject.java

```diff
@@ -58,7 +68,7 @@ public class AuditedObject extends BaseEntity {
 	@Column(columnDefinition = "boolean default false", nullable = false)
 	private Boolean obsolete = false;
 
-	@FullTextField(analyzer = "autocompleteAnalyzer", searchAnalyzer = "autocompleteSearchAnalyzer", valueBridge = @ValueBridgeRef(type = OffsetDateTimeValueBridge.class))
+    @FullTextField(analyzer = "autocompleteAnalyzer", searchAnalyzer = "autocompleteSearchAnalyzer", valueBridge = @ValueBridgeRef(type = OffsetDateTimeValueBridge.class))
```

Comment:
For some reason this is 4 spaces being injected?

Reply:
Was because I fixed a git merge conflict in emacs - sorted now